overlord/devicestate/devicestate_remodel_test.go

```diff
@@ -952,15 +953,15 @@ func (s *deviceMgrRemodelSuite) TestRemodelClashInProgress(c *C) {
 		"kernel":         "pc-kernel",
 		"gadget":         "pc",
 		"base":           "core18",
-		"required-snaps": []interface{}{"new-required-snap-1", "new-required-snap-2"},
+		"required-snaps": []interface{}{"new-required-snap-1"},
```

Comment:
Does the number of snaps here affect number of changes?

Reply:
Yes, the remodel will install these snaps before create the actual remodel change. Each of the installation calls (mocked in line 922) creates a new remodel change with then causes the conflict. Since we there's two changes, the last remodel change can conflict with any of them

Reply:
Got it, thanks for explanation!